loopy/expression.py

```diff
@@ -44,6 +44,8 @@ def dtype_to_type_context(target, dtype):
         return "d"
     if isinstance(dtype, NumpyType) and dtype.dtype in [np.float32, np.complex64]:
         return "f"
+    if isinstance(dtype, NumpyType) and dtype.dtype in [np.bool8, np.bool_]:
+        return "b"
```

Comment:
Want to make type contexts an `enum` (https://docs.python.org/3/library/enum.html) to better document possible values?

Reply:
Sorry, but would be quite intrusive.

Reply:
#260